### PR TITLE
DO NOT MERGE :: wainting on a new release of pyseas

### PR DESCRIPTION
**Pull request by ocefpaf**
@rsignell-usgs who else should be a maintainer here?

### CODE REVIEW

**Comment by rsignell-usgs** [comment]
@ocefpaf, please add @cwingard and @joefutrelle

File: recipes/pyseas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - geomag
+    - gsw
```
Comment (ocefpaf) [inline recipes/pyseas/meta.yaml]: Looking at the code it seems that `pyseas` uses `pygsw` and now `gsw`. Can @cwingard or @joefutrelle confirm that?
If so we need to package `pygsw` in conda-forge. (We do not have it at the moment b/c it is slow (vectorized from c), outdated, and hard to build.)

**Comment by ocefpaf** [comment]
`prs_functions_tides_2014_thru_2019.mat` is almost 40 G is that file making this package huge.
Is there something we can do do reduce the size?

**Comment by cwingard** [comment]
No. The original code from ion-functions used pygsw. I'm converting 
pyseas over to using gsw. I've just started that process. It may be 
premature to set this up in conda-forge?

**Comment by ocefpaf** [comment]
>  It may be premature to set this up in conda-forge?

We can wait for the next release to merge this.

**Pull request closed without merging**